README.md

```diff
@@ -269,7 +269,7 @@ Net-SNMP module support is available with these operating systems:
 
 * RedHat family  - tested on CentOS 7
 * SuSE family    - tested on SLES 11 SP1
-* Debian family  - tested on Debian 9, Debian 10, Debian 11, Debian 12, Ubuntu 18.04, Ubuntu 20.04
+* Debian family  - tested on Debian 11, Debian 12, Debian 13, Ubuntu 20.04, Ubuntu 22.04, Ubuntu 24.04
```

Comment:
It's really annoying to update the list from time to time. I suggest to rephrase it like 'please check metadata.json for all currently tested operating systems'